gnucash/gnucash.cpp

```diff
@@ -296,7 +296,7 @@ main(int argc, char ** argv)
         << "\n"
         // Translators: Do not translate $DISPLAY! It is an environment variable for X11
         << _("Error: could not initialize graphical user interface and option add-price-quotes was not set.\n"
-        "Perhaps you need to set the $DISPLAY environment variable?");
+        "Perhaps you need to set the $DISPLAY environment variable?\n");
```

Comment:
why not just `") << "\n"`

Reply:
weird workaround instead of correct solution, breaks some existing translations

Reply:
ok

Reply:
What's weird about @chloekek 's suggestion ? That's perfectly normal c++ and it avoids having to add coded newline characters in translations. Note how line 296 does exactly the same thing.

As for breaking existing translations, so does your proposal :wink: albeit the exact opposite set of translations. You even added fixes for them in your PR.

I'd be tempted to go with @chloekek 's suggestion.